src/server.py

```diff
@@ -104,4 +104,155 @@ async def discard_cards(deck_id: str, cards: str):
     
     return {"message": "Cards discarded successfully"}
 
+
+
+@app.get("/api/v2/highestRank/~{cards}")
```

Comment:
What does the `~` do in the route here?

Reply:
I was getting a certain error and that seemed to fix the problem, might have been a different change that fixed the problem but I don't remember tbh.